setup.py

```diff
@@ -41,22 +41,23 @@
         'Programming Language :: Python :: 3.8',
         'Programming Language :: Python :: 3.9',
     ],
-    test_suite='tests',
-    setup_requires=[
-        # dependency for `python setup.py test`
-        'pytest-runner',
-        # dependencies for `python setup.py build_sphinx`
-        'sphinx',
-        'recommonmark',
-        'sphinx_rtd_theme',
-    ],
-    tests_require=[
-        'deepdiff',
-        'pytest',
-        'pytest-cov',
-        'pycodestyle',
-    ],
     extras_require={
-        'dev':  ['prospector[with_pyroma]', 'yapf', 'isort'],
+        'dev':  [
+            # Test
+            'deepdiff',
+            'pytest',
+            'pytest-cov',
+            # Linters
+            'pycodestyle',
+            'prospector[with_pyroma]',
+            'yapf',
+            'isort',
+            # dependency for `pytest`
+            'pytest-runner',
+            # dependencies for `cd docs && make html`
```

Comment:
I could not find the command `cd docs && make html`  in either contributing or documentation. I could build the documentation using the command `python setup.py build_sphinx -Ea`.  I suggest removing the command from the comment here.

Reply:
Added line to CONTRIBUTING.rst and simplified comment in setup.py.